circle.yml

```diff
@@ -56,6 +56,10 @@ test:
         parallel: true
     - rsync -uav "build/reports/profile" "$CIRCLE_ARTIFACTS/":
         parallel: true
+    - mkdir -p $CIRCLE_ARTIFACTS/heapdumps:
```

Comment:
That's not what `parallel` does. `parallel` just runs it on all containers. `background` does what you're thinking of (see higher up in the `circle.yml`)